lib/Doctrine/CouchDB/CouchDBClient.php

```diff
@@ -143,15 +143,39 @@ public function getUuids($count = 1)
     }
 
     /**
-     * Find a document by ID and return the HTTP response.
+     * Find a document by ID and return the HTTP response. If
+     * $getAllLeafRevisions is true, documents for all leaf revisions are
```

Comment:
Nit-pick; I'd leave a full empty line between "Find a document by ID and return the HTTP response." and the detailed description. So it would be:

```php
/**
 * Find a document by ID and return the HTTP response.
 *
 * If $getAllLeafRevisions is true...
 */
```